### PR TITLE
[#159022583]: View condition notes on manage assets endpoint

### PR DESCRIPTION
**Pull request by nelmandela**
### What does this PR do?

  •Enables user to see condition notes on manage-assets endpoint

### Description of task to be completed:

  •Refactor` asset_condition` to `condition_notes` field
  •Add `condition_notes` field to asset serializer
  •Rafactor tests

### How can this be manually tested?

  •`http://127.0.0.1:8000/api/v1/docs/` make a request to the manage-assets endpoint

### Relevant pivotal tracker stories:

  •[#159022583](https://www.pivotaltracker.com/story/show/159022583)

### Screenshot

screen shot 2018-07-24 at 15 35 33 2

### CODE REVIEW

**Comment by collinmutembei** [comment]
What does this post_save signal achieve?
https://github.com/AndelaOSP/art-backend/blob/a919aa370b8c94abf651202bec7690b500f614bc/core/models/asset.py#L479-L487

File: core/models/asset.py [hunk 1]
```diff
@@ -257,7 +257,7 @@ class Asset(models.Model):
                                      null=True,
                                      on_delete=models.PROTECT)
     current_status = models.CharField(editable=False, max_length=50)
-    asset_condition = models.CharField(editable=False,
+    condition_notes = models.CharField(editable=False,
```
Comment (egichuri) [inline core/models/asset.py]: while we're here.. can we remove this default value?

Comment (egichuri) [reply core/models/asset.py]: also, let's use `asset_condition_notes` all through

Comment (collinmutembei) [reply core/models/asset.py]: @nelmandela To ensure that notes on the condition of an asset are not limited to 50 chars, prefer TextField over CharField.

**Comment by collinmutembei** [comment]
Let's leave the post_save signal receiver as is for now, we'll refactor this later.

**Comment by nelmandela** [comment]
@collinmutembei @egichuri

**Pull request merged**